available-hardware.mdx

```diff
@@ -3,10 +3,10 @@ title: "Available Hardware"
 description: "A list of hardware that is available on Cerebrium's platform."
 ---
 
-The Cerebrium platform allows you to quickly and easily fine-tune and deploy machine learning workloads on a variety of different hardware.
+The Cerebrium platform allows you to quickly and easily deploy machine learning workloads on a variety of different hardware.
 We take care of all the hard work so you don't have to. Everything from the hardware drivers to the scaling of your deployments is managed by us so that you can focus on what matters most: your use case.
 
-This page lists the hardware that is currently available on the platform.
+This page lists the hardware that is currently available on the platform. If you would like us to support additional chips on the platform please reach out to [Support](mailto:support@cerebrium.ai)
```

Comment:
Chips sounds more like we're talking about cpus. Maybe additional cards? hardware?